test/concurrency/testing_transaction_util.cpp

```diff
@@ -456,5 +474,83 @@ bool TestingTransactionUtil::ExecuteScan(
   }
   return true;
 }
+
+ResultType TestingTransactionUtil::UpdateTuple(storage::DataTable *table,
+                                               const int key) {
+  srand(15721);
```

Comment:
Is it ok to have non-deterministic stuff in the test cases? Shouldn't they be reproducible at any chance?